File: src/main/java/fr/bfr/api/DropRepository.java
package fr.bfr.api;

import fr.bfr.model.DropChance;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.lang.NonNull;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public interface DropRepository extends JpaRepository<DropChance, Integer> {

    @NonNull
    List<DropChance> findAll();
}

File: src/main/kotlin/fr/bfr/api/DropRepository.kt
package fr.bfr.api

import fr.bfr.model.DropChance
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface DropChanceRepository : JpaRepository<DropChance, Int> {
    override fun findAll(): List<DropChance>
}
